src/app/counter/Display/display.css

```diff
@@ -0,0 +1,29 @@
+.display {
```

Comment:
use .module.css files instead of .css files to avoid name conflicts in the future